tools/StaticAnalysis/Exceptions/Az.Network/SignatureIssues.csv

```diff
@@ -1,420 +1,421 @@
+Microsoft.Azure.PowerShell.Cmdlets.Network.dll,Microsoft.Azure.Commands.Network.RemoveAzureLoadBalancerCommand,Remove-AzLoadBalancer,1,8600,Cmdlet 'Remove-AzLoadBalancer' has no defined output type.,"Add an OutputType attribute that declares the type of the object(s) returned by this cmdlet. If this cmdlet returns no output, please set the output type to 'bool' and make sure to implement the 'PassThru' parameter."
+Microsoft.Azure.PowerShell.Cmdlets.Network.dll,Microsoft.Azure.Commands.Network.SetAzureLoadBalancerCommand,Set-AzLoadBalancer,1,8100,Set-AzLoadBalancer Does not support ShouldProcess but the cmdlet verb Set indicates that it should.,Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue
+Microsoft.Azure.PowerShell.Cmdlets.Network.dll,Microsoft.Azure.Commands.Network.RemoveAzureNetworkInterfaceCommand,Remove-AzNetworkInterface,1,8600,Cmdlet 'Remove-AzNetworkInterface' has no defined output type.,"Add an OutputType attribute that declares the type of the object(s) returned by this cmdlet. If this cmdlet returns no output, please set the output type to 'bool' and make sure to implement the 'PassThru' parameter."
+Microsoft.Azure.PowerShell.Cmdlets.Network.dll,Microsoft.Azure.Commands.Network.SetAzureNetworkInterfaceCommand,Set-AzNetworkInterface,1,8100,Set-AzNetworkInterface Does not support ShouldProcess but the cmdlet verb Set indicates that it should.,Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue
+Microsoft.Azure.PowerShell.Cmdlets.Network.dll,Microsoft.Azure.Commands.Network.GetAzureNetworkSecurityGroupCommand,Get-AzNetworkSecurityGroup,1,8510,"Cmdlet 'Get-AzNetworkSecurityGroup' has multiple parameter sets, but no defined default parameter set.",Define a default parameter set in the cmdlet attribute.
+Microsoft.Azure.PowerShell.Cmdlets.Network.dll,Microsoft.Azure.Commands.Network.NewAzureNetworkSecurityRuleConfigCommand,New-AzNetworkSecurityRuleConfig,1,8100,New-AzNetworkSecurityRuleConfig Does not support ShouldProcess but the cmdlet verb New indicates that it should.,Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue
+Microsoft.Azure.PowerShell.Cmdlets.Network.dll,Microsoft.Azure.Commands.Network.GetAzureNetworkSecurityRuleConfigCommand,Get-AzNetworkSecurityRuleConfig,1,8410,Parameter DefaultRules of cmdlet Get-AzNetworkSecurityRuleConfig does not follow the enforced naming convention of using a singular noun for a parameter name.,Consider using a singular noun for the parameter name.
+Microsoft.Azure.PowerShell.Cmdlets.Network.dll,Microsoft.Azure.Commands.Network.RemoveAzureNetworkSecur
```

Comment:
why there are so many lines changed. There should only be 1 line added

Reply:
It's just the addition of 1 line only. Rest of the contents are unchanged.

It's just addition/deletion of the same lines.